src/store/authStore.ts

```diff
@@ -21,14 +22,16 @@ const authState = reactive<AuthState>({
 // @TODO get name from backend not URL
 export const loginName = window.location.host.split('.')[0];
 
-export const myYggdrasilAddress = async () => {
-    const res = await axios.get(`${window.location.origin}/api/v2/locations/yggdrasil`);
-    return res.data;
-};
+const myLocation = ref<string>('');
+// export const myYggdrasilAddress = async () => {
+//     const res = await axios.get(`${window.location.origin}/api/v2/locations/yggdrasil`);
+//     return res.data;
+// };
```

Comment:
For every line of code in comments a seal dies, this is true, many people know this
![image](https://user-images.githubusercontent.com/6184820/168887685-5fd3ab97-1f85-4367-969d-939d9bf1f382.png)